source/elements/oneDAL/source/data_management/index.rst

```diff
@@ -5,12 +5,140 @@
 Data management
 ===============
 
-The Data Management component includes classes and utilities for data acquisition, initial preprocessing
-and normalization, for data conversion into numeric formats (performed by one of supported Data Sources)
-and for model representation.
+Effective data management is among key constituents of the performance of a data
+analytics application. For |dal_short_name|, effective data management requires
+effectively performing the following operations:
+
+- raw data acquisition
+- filtering and normalization
+- data conversion to a numeric representation
+- data streaming to an algorithm
+
+Key concepts
+============
+
+|dal_short_name| provides a set of customizable interfaces to operate on
+out-of-memory and in-memory data in different usage scenarios, which include
+:term:`batch processing <Batch mode>`, :term:`online processing <Online mode>`,
+and complex scenarios.
+
+Table
+-----
+
+To represent a :term:`dataset` in a numeric :term:`data format`,
+|dal_short_name| introduces a :term:`table` concept - an :term:`immutable
+<Immutability>` tabular view of a :term:`dataset` where table rows represent
+:term:`observations <Observation>` and columns represent :term:`features
+<Feature>`.
+
+.. image:: _static/table.png
+  :width: 400
+  :alt: Table
+
+|dal_short_name| defines a set of table types, each represent data in
+various :term:`data formats <Data format>`. See :ref:`tables` section for more
+details about requirements of table types and table concept.
+
+Data source
+-----------
+
+Data source defines an interface to extract data from raw sources like files,
+databases, etc. and to represent them as a :term:`table` object. It can
+automatically transform non-numeric :term:`categorical <Categorical feature>`
+and :term:`continuous <Continuous feature>` values of data into a numeric
+representation.
+
+It can also provide an abilities to filter or normalize :term:`feature` values
+being extracted or recover missing values:
+
+- It is not a rare situation when only a subset of features can be measured at a
+  given moment. In this case, the non-measured features in the feature vector
+  become blank, or missing. Special statistical techniques enable recovery
+  (emulation) of missing values.
+
+- When a given measurement introduces an :term:`outlier` or when an observed
+  object can produce an :term:`outlier` by nature, special statistical techniques
+  enable detection of outliers and recovery of the abnormal data.
+
+Data are not always fit the local memory, especially when processing with
+accelerators. Data source provides an interface to load data by batches and
+orchestrates its efficient extraction directly into local memory of the device.
+Therefore data source enables complex scenarios of data analytics, like
+:term:`online computations <Online mode>`.
+
+Table builder
+-------------
+
+Some :term:`table` objects require complex procedures to create them. To make
```

Comment:
The same comment about the style. Instead of setting context on 'some table requires ..' start with formal precise def which generally gives additional info/more details about builder. Explain what problem it solves using precise but still high level lang, etc. Look at the text from the perspective of the reader who sees this text for the first time and wants to get a clear understanding about the builder  after reading this section. The same comment applies to other sections